src/Guzzle/RemoveValidationMiddleware.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace eLife\Journal\Guzzle;
+
+use function GuzzleHttp\Psr7\parse_query;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+final class RemoveValidationMiddleware
```

Comment:
the name of the class suggests this is a generic middleware that could be reused, but it isn't, so a more specific name would reflect that (not worth it to make it configurable)